Makefile.system

```diff
@@ -606,7 +606,6 @@ ifneq ($(C_COMPILER), GCC)
 DYNAMIC_CORE += POWER9
 endif
 ifeq ($(C_COMPILER), GCC)
-GCCVERSIONGT5 := $(shell expr `$(CC) -dumpversion | cut -f1 -d.` \> 5)
```

Comment:
@mhillenibm This is needed for POWER as the variable was set under `ifeq ($(OSNAME), WINNT)` in the previous lines..

Reply:
Ok.. they are not arch specific now.. This is fine then .

Reply:
it was actually rescued out of the WINNT-only section by #2626 so de-duplicating should be ok

Reply:
note that the definition has moved out of the OS-specific section, in #2626